components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/RegistryPersistenceImpl.java

```diff
@@ -1071,10 +1071,6 @@ private DevPortalAPISearchResult searchPaginatedDevPortalAPIs(Registry userRegis
                     }
                 }
                 apiInfo.setAvailableTierNames(availableTiers);
-                apiInfo.setSubscriptionAvailability(
-                        artifact.getAttribute(APIConstants.API_OVERVIEW_SUBSCRIPTION_AVAILABILITY));
-                apiInfo.setSubscriptionAvailableOrgs(
-                        artifact.getAttribute(APIConstants.API_OVERVIEW_SUBSCRIPTION_AVAILABLE_TENANTS));
```

Comment:
any reason for removing these? we shouldn't modify any in RegistryPersistenceImpl.java  class